Bug fix for teams page.

BradHN1 opened the pull request:
Need to merge to dev and update canary

frimpongopoku reviewed (commented):
Hi @BradHN1 , I see you mostly replaced `teamsPage` with 'teams` but what was the bug? What were you trying to fix?

BradHN1 commented:
The bug was that when I added the customization of teams page settings, the name teamsPage was used both for the page customization and the teams info.  I changed it so that the teams info used a different name.  
I;ve added a corresponding issue: https://github.com/massenergize/frontend-portal/issues/547


> On Apr 21, 2021, at 11:19 PM, Frimpong Opoku Agyemang ***@***.***> wrote:
> 
> 
> @frimpongopoku commented on this pull request.
> 
> Hi @BradHN1 <https://github.com/BradHN1> , I see you mostly replaced teamsPage with 'teams` but what was the bug? What were you trying to fix?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/massenergize/frontend-portal/pull/546#pullrequestreview-641744787>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ADBVDZFRWXOTXROV3B2GCA3TJ6IS7ANCNFSM43LNU7KA>.
>
